frontend/src/scenes/funnels/FunnelViz.tsx

```diff
@@ -151,6 +153,20 @@ export function FunnelViz({
                     dashboardItemId={dashboardItemId || fromItem}
                     inSharedMode={inSharedMode}
                     percentage={true}
+                    onClick={
+                        dashboardItemId
+                            ? null
```

Comment:
Seems a little unclear to pass `null` here rather than a function that either calls `loadPeople` or does nothing